### PR TITLE
Look up module and form id from build

### PR DESCRIPTION
**Pull request by esoergel**
http://manage.dimagi.com/default.asp?182456
@czue @sravfeyn

### CODE REVIEW

File: corehq/apps/reports/views.py [hunk 1]
```diff
@@ -1358,6 +1358,23 @@ def download_form(request, domain, instance_id):
     return couchforms_views.download_form(request, instance_id)
 
 
+def _form_instance_to_context_url(domain, instance):
+    try:
+        build = Application.get(instance.build_id)
+    except ResourceNotFound:
+        raise Http404(_('Application not found.'))
+
+    for module in build.get_modules():
```
Comment (snopoke) [inline corehq/apps/reports/views.py]: You can use `get_form_by_xmlns`: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/models.py#L4863-4863

**Comment by czue** [comment]
:+1: from me when feedback is addressed

**Comment by sravfeyn** [comment]
Merging this because the ticket needs quick resolution. Hoping you can respond to feedback in another PR. @esoergel

File: corehq/apps/reports/views.py (hunk 1 above)
Comment (esoergel) [inline corehq/apps/reports/views.py]: https://github.com/dimagi/commcare-hq/pull/8313

**Pull request merged**